### PR TITLE
Add explanation, update example, minor fixes

### PR DESCRIPTION
**Pull request by alijnmerchant21**
- Added basic (elementary) explanation of order book, buy order and sell order
 - Added a simpler example (Simplified the calculations by using coherent values for a token and integer amounts) to "Performing an exchange with Sell order." (Original example is still kept as a comment for reference) 
- Used the term 'marscoin' and 'venuscoin' instead of mcx and vcx respectively
- Minor spelling and language fixes

### CODE REVIEW

File: interchain-exchange/tutorial/00-tutorial.md [hunk 1]
```diff
@@ -16,23 +16,42 @@ If you want to see the end result, please refer to the [example implementation](
 - Send IBC packets from one blockchain to another
 - Deal with timeouts and acknowledgements of IBC packets
 
+## Concepts good to know
+
+**Order Books**
+
+**Order books** are electronic list of *buy* and *sell* orders for specific financial instrument - For eg: cryptocurrencies.
+An order book lists the number of tokens being *bid* (buy) on or *offered* (sell) at each price point, or market depth. It also identifies the market participants behind the buy and sell orders. These lists help traders and also improve market transparency because they provide valuable trading information.
```
Comment (fadeev) [inline interchain-exchange/tutorial/00-tutorial.md:24]:
```suggestion
**Order books** are electronic list of *buy* and *sell* orders for specific financial instrument - For eg: cryptocurrencies. An order book lists the number of tokens being *bid* (buy) on or *offered* (sell) at each price point, or market depth. It also identifies the market participants behind the buy and sell orders. These lists help traders and also improve market transparency because they provide valuable trading information.
```

File: interchain-exchange/tutorial/00-tutorial.md [hunk 2]
```diff
@@ -16,23 +16,42 @@ If you want to see the end result, please refer to the [example implementation](
...
+The module allows to open an exchange order book between a pair of token within a blockchain network and between a pair of token on another blockchain network. The blockchains are required to have the `ibcdex` module available.
 
 Tokens can be bought or sold with Limit Orders on a simple order book, there is no notion of Liquidity Pool or AMM.
 
 The market is unidirectional: the token sold on the source chain cannot be bought back as it is, and the token bought from the target chain cannot be sold back using the same pair. If a token on a source chain is sold, it can only be bought back by creating a new pair on the order book. This is due to the nature of IBC, creating a `voucher` token on the target blockchain. In this tutorial you will learn the difference of a native blockchain token and a `voucher` token that is minted on another blockchain. You will learn how to create a second order book pair in order to receive the native token back.
 
+<!-- The above comment on line 27 is a bit hard to follow. Adding an example below for easier understanding.-->
```
Comment (fadeev) [inline interchain-exchange/tutorial/00-tutorial.md:44]:
```suggestion
```

Comments are visible to users, let's not add them, unless they are meant for end-users.

File: interchain-exchange/tutorial/00-tutorial.md [hunk 3]
```diff
@@ -16,23 +16,42 @@ If you want to see the end result, please refer to the [example implementation](
...
+
+For example: There are two `ibc` enabled chains - Earth_chain and Venus_chain.
```
Comment (fadeev) [inline interchain-exchange/tutorial/00-tutorial.md:46]:
```suggestion
For example: There are two IBC-enabled chains - Earth chain and Venus chain. 
```

Earth_chain is confusing.

File: interchain-exchange/tutorial/00-tutorial.md [hunk 4]
```diff
@@ -16,23 +16,42 @@ If you want to see the end result, please refer to the [example implementation](
...
 
 # App Design
 
-In this chapter you will learn how the interchain exchange module is designed. The module has order books, buy- and sell orders. First, an order book for a pair of token has to be created. After an order book exists, you can create buy and sell orders for this pair of token.
+In this chapter you will learn how the interchain exchange module is designed. The module has **order books**, **buy** and **sell orders.** First, an order book for a pair of token has to be created. After an order book exists, you can create buy and sell orders for this pair of token.
 
-The module will make use of the Inter-Blockchain Communication Standard [IBC](https://github.com/cosmos/ics/blob/master/ibc/2_IBC_ARCHITECTURE.md). With use of the IBC, the module can create order books for tokens to have multiple blockchains interact and exchange their tokens. 
+The module will make use of the Inter-Blockchain Communication Standard [IBC](https://github.com/cosmos/ics/blob/master/ibc/2_IBC_ARCHITECTURE.md). With use of the IBC, the module can create order books for tokens to have multiple blockchains interact and exchange their tokens. <!-- The link provided gives a 404. -->
```
Comment (fadeev) [inline interchain-exchange/tutorial/00-tutorial.md:54]:
```suggestion
The module will make use of the Inter-Blockchain Communication Standard [IBC](https://ibcprotocol.org/). With the use of IBC, the module can create order books for tokens to have multiple blockchains interact and exchange their tokens.
```

File: interchain-exchange/tutorial/00-tutorial.md [hunk 5]
```diff
@@ -388,6 +410,51 @@ balances:
...
 
+```yml
+# Target blockchain
+balances:
+- amount: "5" # increased from 0
+  denom: ibc/99678A10AF684E33E88959727F2455AE42CCC64CD76ECFA9691E1B5A32342D33 # mcx voucher
+```
+-->
+
+We now have two orders open for `marscoin`: a *sell order* on the **source chain** (for 20 `marscoin` at 5 `venuscoin`) and a *buy order* on the **target chain** (for 5 `marscoin` at 5 `venuscoin`). Let's perform an exchange by sending a sell order to the source chain.
```
Comment (fadeev) [inline interchain-exchange/tutorial/00-tutorial.md:421]: If we switch to `marscoin`, we need to change the denom in config as well as in examples.

Comment (alijnmerchant21) [reply interchain-exchange/tutorial/00-tutorial.md:419]: We can still use mcx and vcx while writing code. But for explanation I propose on using `marscoin` and `venuscoin` respectively.

Comment (fadeev) [reply interchain-exchange/tutorial/00-tutorial.md:419]: Arguably, this can confuse people, because we use `code` syntax, so either 1. `marscoin` everywhere, or 2. `mcx` everywhere, 3. or `mcx` and Mars coin.

Comment (barriebyron) [reply interchain-exchange/tutorial/00-tutorial.md:419]: I vote for same in code and text, and yes, using the tick marks for code syntax for literals

File: interchain-exchange/tutorial/00-tutorial.md [hunk 6]
```diff
@@ -388,6 +408,51 @@ balances:
...
+```yml
+# Source blockchain
+balances:
+- amount: "25" # increased from 0
+  denom: ibc/50D70B7748FB8AA69F09114EC9E5615C39E07381FE80E628A1AF63A6F5C79833 # vcx voucher
+- amount: "985" # decreased from 990
+  denom: mcx
+```
+
+The counterparty (sender of the buy mcx order) received 5 `marscoin` vouchers. `venuscoin` balance hasn't changed, because the correct amount of `venuscoin` (50) were locked at the creation of the buy order during the previous step.
```
Comment (barriebyron) [inline interchain-exchange/tutorial/00-tutorial.md:454]: do we mean mcx here? Or did we decide to spell out marscoin?

File: interchain-exchange/tutorial/00-tutorial.md [hunk 7]
```diff
@@ -577,7 +642,7 @@ Buy order book on the target blokchain is now empty.
 
 In this chapter you will implement the logic for creating order books.
```
Comment (barriebyron) [inline interchain-exchange/tutorial/00-tutorial.md:643]:
```suggestion
In this section, you implement the logic for creating order books.
```

File: interchain-exchange/tutorial/00-tutorial.md [hunk 8]
```diff
@@ -577,7 +642,7 @@ Buy order book on the target blokchain is now empty.
...
 
-In Cosmos SDK the state is stored in a key-value store. Each order book will be stored under a unique key composed of four values: port ID, channel ID, source denom and target denom. For example, an order book for `mcx` and `vcx` could be stored under `ibcdex-channel-4-mcx-vcx`. Define a function that returns an order book store key.
+In Cosmos SDK the state is stored in a key-value store. Each order book will be stored under a unique key composed of four values: port ID, channel ID, source denom and target denom. For example, an order book for `mcx` (`marscoin`) and `vcx` (`venuscoin`) could be stored under `ibcdex-channel-4-mcx-vcx`. Define a function that returns an order book store key.
```
Comment (barriebyron) [inline interchain-exchange/tutorial/00-tutorial.md:645]:
```suggestion
In Cosmos SDK, the state is stored in a key-value store. Each order book is stored under a unique key that is composed of four values: port ID, channel ID, source denom, and target denom. 

For example, an order book for `mcx` (`marscoin`) and `vcx` (`venuscoin`) could be stored under `ibcdex-channel-4-mcx-vcx`. Define a function that returns an order book store key.
```

Comment (barriebyron) [reply interchain-exchange/tutorial/00-tutorial.md:645]: let's focus on present tense, even with tutorials, I know we struggle with this in all of our tutorials, see https://developers.google.com/style/tense

**Review by barriebyron — COMMENTED** [review]
@alijnmerchant21 I love seeing this PR and all of the work that you are putting into helping our users! So awesome, be proud. Since it's our first editorial review for a PR, I provided granular feedback and editorial suggestions. lmk if you have questions, I am here to serve and love doing this type of feedback work.

File: interchain-exchange/tutorial/00-tutorial.md (hunk 6 above)
Comment (alijnmerchant21) [reply interchain-exchange/tutorial/00-tutorial.md:449]: I suggest to keep `marscoin` and `venuscoin` when explaining. For code we can use mcx and vcx

File: interchain-exchange/tutorial/00-tutorial.md [hunk 9]
```diff
@@ -16,23 +16,35 @@ If you want to see the end result, please refer to the [example implementation](
...
+
 In the next chapter you will learn details about the design of the interblockchain exchange.
 
 # App Design
 
-In this chapter you will learn how the interchain exchange module is designed. The module has order books, buy- and sell orders. First, an order book for a pair of token has to be created. After an order book exists, you can create buy and sell orders for this pair of token.
+This section describes how the interchain exchange module is designed. The module has order books, buy orders, and sell orders. First, an order book for a pair of tokens must be created. After an order book exists, you can create buy and sell orders for this pair of tokens.
 
-The module will make use of the Inter-Blockchain Communication Standard [IBC](https://github.com/cosmos/ics/blob/master/ibc/2_IBC_ARCHITECTURE.md). With use of the IBC, the module can create order books for tokens to have multiple blockchains interact and exchange their tokens. 
+The module will make use of the Inter-Blockchain Communication Standard [IBC](https://ibcprotocol.org/). With the use of IBC, the module can create order books for tokens to have multiple blockchains interact and exchange their tokens.
```
Comment (barriebyron) [inline interchain-exchange/tutorial/00-tutorial.md:47]:
```suggestion
The module will make use of the Inter-Blockchain Communication protocol [IBC](https://ibcprotocol.org/). With the use of IBC, the module can create order books for tokens to have multiple blockchains interact and exchange their tokens.
```

File: interchain-exchange/tutorial/00-tutorial.md [hunk 10]
```diff
@@ -351,7 +364,9 @@ BuyOrderBook:
 
 ## Performing an Exchange with a Sell Order
 
-We now have two orders open for MCX: a sell order on the source chain (for 10mcx at 15vcx) and a buy order on the target chain (for 5mcx at 5vcx). Let's perform an exchange by sending a sell order to the source chain.
+<!-- This is a bit difficult to follow. Using a simpler example to illustrate exchange with sell order. Simplifying the calculations by using coherent values for a token and integer amount -->
```
Comment (barriebyron) [inline interchain-exchange/tutorial/00-tutorial.md:367]: before merging, let's delete all comments in the Markdown files because these comments are visible to all since our repo is open source, thanks @alijnmerchant21

File: interchain-exchange/tutorial/00-tutorial.md [hunk 11]
```diff
@@ -388,6 +403,51 @@ balances:
 
 The counterparty (sender of the buy mcx order) received 5 mcx vouchers. vcx balance hasn't changed, because the correct amount of vcx (50) were locked at the creation of the buy order during the previous step.
 
+```yml
+# Target blockchain
+balances:
+- amount: "5" # increased from 0
+  denom: ibc/99678A10AF684E33E88959727F2455AE42CCC64CD76ECFA9691E1B5A32342D33 # mcx voucher
+```
+-->
```
Comment (barriebyron) [inline interchain-exchange/tutorial/00-tutorial.md:412]:
```suggestion
```

Comment (barriebyron) [reply interchain-exchange/tutorial/00-tutorial.md:412]: we probably want to delete this end comment tag?

File: interchain-exchange/tutorial/00-tutorial.md (hunk 6 above)
Comment (barriebyron) [reply interchain-exchange/tutorial/00-tutorial.md:449]: oooh I know it's work, but I'm in agreement with Denis on this... is there a way to use consistent token names and denom? Teach me!

**Review by barriebyron — COMMENTED** [review]
let me know if I can help @alijnmerchant21 I am learning, too!

**Comment by barriebyron** [comment]
hi @alijnmerchant21 lmk if you need my attention here for any details. 
I made suggestions inline, you can commit a suggestion one at a time (noisy) or commit suggestions in batch (happier). You can even edit a suggestion. I love the suggesting feature but have discovered not all of us are familiar with it.
I'm off until August 9 but can check back here periodically.

**Comment by toschdev** [comment]
The `mcx` and `vcx` token were not updated consistently. Since the PR is open for a long time, I am not expecting new changes to this one. Please reopen when you start working on it again @alijnmerchant21
